xayn-ai-ffi-c/src/reranker/ai.rs

```diff
@@ -400,6 +481,21 @@ mod tests {
         }
     }
 
+    struct TestSyncdata(CBytes);
+
+    impl Default for TestSyncdata {
+        fn default() -> Self {
+            Self(vec![0; 17].into_boxed_slice().into())
+        }
+    }
+
+    impl TestSyncdata {
+        #[allow(clippy::clippy::unnecessary_wraps)]
```

Comment:
seems like the autocompletion added a double clippy
```suggestion
        #[allow(clippy::unnecessary_wraps)]
```

Reply:
thanks, changed it. funny it still worked fine..